doc/cephadm/index.rst

```diff
@@ -40,7 +40,7 @@ The ``cephadm`` utility is used to bootstrap a new Ceph Cluster.
 
 Use curl to fetch the standalone script::
 
-  [monitor 1] # curl --silent --remote-name --location https://github.com/ceph/ceph/raw/master/src/cephadm/cephadm
+  [monitor 1] # curl --silent --remote-name --location https://github.com/ceph/ceph/raw/octopus/src/cephadm/cephadm
```

Comment:
interesting, that we don't have a [`res_epilog`](https://www.sphinx-doc.org/en/1.8/usage/configuration.html#confval-rst_epilog)




```suggestion
  [monitor 1] # curl --silent --remote-name --location https://github.com/ceph/ceph/raw/|release_codename|/src/cephadm/cephadm
```

Reply:
to be honest i'm a bit less nervous about maintaing this manually.  for the master branch docs, for instance, i think we should still point to octopus.  (note that you get the master docs when you go to docs.ceph.com.)